experiments/capture-build-scan-info.gradle

```diff
@@ -0,0 +1,49 @@
+import org.gradle.util.GradleVersion
+
+initscript {
+    def pluginVersion = "3.6"
+
+    repositories {
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath("com.gradle:gradle-enterprise-gradle-plugin:${pluginVersion}")
+    }
+}
+
+def isTopLevelBuild = gradle.getParent() == null
+
+if (isTopLevelBuild) {
+    def version = GradleVersion.current().baseVersion
+    def atLeastGradle5 = version >= GradleVersion.version("5.0")
+    def atLeastGradle6 = version >= GradleVersion.version("6.0")
+    
+    if (atLeastGradle6) {
+        settingsEvaluated {
+            it.extensions["gradleEnterprise"].with {
+               buildScan {
+                   buildScanPublished { scan ->
+                       logScan(file("scans.csv"), scan)
+                   }
+               }
+            }
+        }
+    } else if (atLeastGradle5) {
+        projectsEvaluated {
+            rootProject {
+               it.extensions["buildScan"].with {
```

Comment:
```suggestion
               buildScan {
```

Reply:
When I tried this one, it actually worked. Maybe because the closer is invoked within the context of the root project. However, I think I'm going to keep it the way it is just so that this block is more consistent with the Gradle 6 block.